test/dlc_tests/ec2/pytorch/training/test_pytorch_training.py

```diff
@@ -171,6 +172,24 @@ def test_pytorch_amp(pytorch_training, ec2_connection, gpu_only, ec2_instance_ty
     execute_ec2_training_test(ec2_connection, pytorch_training, PT_AMP_CMD)
 
 
+@pytest.mark.integration("pt_s3_gpu")
```

Comment:
nit: Please make this more detailed, like "pt_s3_plugin", since it tests the "S3 Plugin". Similarly, please also make the `integration` marker more clear for the CPU test as well.

Reply:
done